src/main/java/uk/gov/hmcts/reform/professionalapi/infrastructure/controllers/SearchController.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.professionalapi.infrastructure.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.gov.hmcts.reform.professionalapi.domain.service.ProfessionalUserService;
+import uk.gov.hmcts.reform.professionalapi.infrastructure.controllers.response.ProfessionalUserResponse;
+
+@RequestMapping(
+        path = "/search",
+        produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+@RestController
+public class SearchController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OrganisationController.class);
+
```

Comment:
We've been using @SL4J to inject the logger

Reply:
Not even using the logger in that class. Removed